M2/Macaulay2/packages/SimpleDoc/doc.txt

```diff
@@ -1,4 +1,21 @@
 -- -*- mode: M2; coding: utf-8 -*-
+Node
+  Key
+    SimpleDoc
+  Headline
+    simpler documentation for functions and methods
+  Description
+    Text
+      This package provides a human-readable serialization format for
+      documentation nodes as an alternative to the classic way of
+      @TO "writing documentation"@ using the @TO document@ method.
```

Comment:
Make this link work with : 
```
--- a/M2/Macaulay2/packages/SimpleDoc/doc.txt
+++ b/M2/Macaulay2/packages/SimpleDoc/doc.txt
@@ -8,14 +8,14 @@ Node
     Text
       This package provides a human-readable serialization format for
       documentation nodes as an alternative to the classic way of
-      @TO "writing documentation"@ using the @TO document@ method.
+      @TO "Macaulay2Doc::writing documentation"@ using the @TO document@ method.
       The new format is parsed by either @TO doc@ for writing a single node,
       or @TO multidoc@ for writing multiple documentation nodes in one file
       and adheres off-side rule; i.e., using indentation to separate blocks.
   Contributors
     Andrew Hoefel contributed to this package.
   SeeAlso
-    "writing documentation"
+    "Macaulay2Doc::writing documentation"
 Node
  Key
   multidoc
```

Reply:
I thought I tried this, I wonder why it didn't work.